style.css

```diff
@@ -25,70 +41,77 @@ section {
 }
 
 h2 {
+  font-size: 2rem;
   margin-bottom: 20px;
 }
 
-/* ----------App.js---------- */
+/* ----------cyf---------- */
 
-.App {
-  background-color: #f0f0f0;
+body {
+  background-color: var(--gray);
   min-height: 100vh;
+  min-width: 300px;
 }
 
-.App .cyf {
-  background-color: #fff;
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+.cyf {
+  background-color: var(--white);
+  box-shadow: 0 2px 4px var(--shadow);
   border-radius: 5px;
-  margin-top: 20px;
-  padding: 20px;
+  margin-top: 1rem;
```

Comment:
```suggestion

```

Avoid using margin-top if you can. In general, top margins aren't reliable in CSS and in a more complex sheet they can cause unexpected behaviours.

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_box_model/Mastering_margin_collapsing

Well it's not that unexpected. They collapse! 😉